starpy/manager.py

```diff
@@ -214,7 +214,6 @@ def connectionLost(self, reason):
         self.actionIDCallbacks.clear()
         self.eventTypeCallbacks.clear()
     VERSION_PREFIX = 'Asterisk Call Manager'
-    END_DATA = '--END COMMAND--'
```

Comment:
So, while I'm happy to have starpy support the new format, it also has to support the old format.

The code changes being proposed for AMI are backwards incompatible. As a result, the major version of AMI has to be bumped. However, client libraries - such as starpy - that are used with a variety of Asterisk versions (in this case, 1.8+) have to maintain capatibility with all of those versions.

As it is, I think the handling here will need to be reworked to look for both the original END_DATA as well as the new format.